activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-events/src/main/java/org/activiti/cloud/services/events/listeners/TaskCompletedCommandContextActorProviderEventListener.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.events.listeners;
+
+import org.activiti.api.runtime.shared.security.SecurityContextPrincipalProvider;
+import org.activiti.api.task.runtime.events.TaskCompletedEvent;
+import org.activiti.api.task.runtime.events.listener.TaskRuntimeEventListener;
+import org.activiti.engine.impl.context.Context;
+
+public class TaskCompletedCommandContextActorProviderEventListener
```

Comment:
Same the name seems cumbersome to me, suggestion: TaskCompletedActorProviderEventListener (?)